docs/lif_services_overview.md

```diff
@@ -0,0 +1,291 @@
+
+---
+
+### 🧠 LIF Query Planner API
+
+**WHEN YOU NEED TO...**  
+Intelligently route queries, determine data freshness requirements, and orchestrate data collection from multiple upstream sources.
+
+**THIS SERVICE...**  
+Acts as the query intelligence layer, checking cache availability, consulting the Identity Mapper for additional learner identities, and orchestrating data collection workflows when needed.
+
+**USE CASES:**
+
+- Optimizing query execution across multiple data sources
+- Determining when cached data is sufficient vs. when fresh data is needed
+- Coordinating complex queries that require identity resolution
+- Managing data collection workflows for missing or stale data
+
+**WORKS WITH:** GraphQL API (receives queries), Query Cache API, Identity Mapper API, Orchestrator API  
+**TYPICAL USERS:** Platform engineers, DevOps teams
+
+---
+
+### 🔧 LIF Orchestrator API
+
+**WHEN YOU NEED TO...**  
+Integrate with existing workflow orchestration tools to manage data collection pipelines from upstream source systems.
+
+**THIS SERVICE...**  
+Provides a facade to external orchestration products (Dagster, Apache Airflow), enabling the Query Planner to trigger data collection workflows.
+
+**USE CASES:**
+
+- Scheduling regular data imports from institutional systems
+- Triggering on-demand data collection when cache misses occur
+- Managing complex ETL pipelines that include translation steps
+- Coordinating data freshness across multiple source systems
+
+**WORKS WITH:** Query Planner API (triggers workflows), Translator API (may be included in workflows)  
+**TYPICAL USERS:** Data engineers, DevOps teams, workflow administrators
+
+---
+
+## How These Services Work Together
+
+**Example Query Flow:**
+
+1. A user queries the **GraphQL API** for a learner's complete record
+2. **GraphQL API** forwards the query to **Query Planner API**
+3. **Query Planner API** checks the **Query Cache API** for existing data
+4. If data is missing or stale, **Query Planner API** consults **Identity Mapper API** for all known identities
+5. **Query Planner API** triggers workflows via **Orchestrator API** to collect data
+6. Orchestrator workflows fetch data from source systems, using **Translator API** if needed
+7. Collected data is stored in **Query Cache API** as fragments and merged into an aggregate record
+8. **GraphQL API** returns the complete learner record to the user
+
+**Example AI-Powered Use Case:**
+
+1. A student opens the **Advisor UI** and asks a question
+2. **Advisor UI** sends the question to **Advisor API**
+3. **Advisor API** queries learner data through **GraphQL API**
+4. The query flows through the standard infrastructure (Query Planner, Cache, etc.)
+5. **Advisor API** processes the data with AI and returns a natural language response
+6. **Advisor UI** displays the conversational answer to the student
+
+---
+
+## Composing Solutions
+
+Not every organization needs all services. Here are common deployment patterns:
```

Comment:
This makes it sound like LIF is deployed with enough adopters for patterns to emerge. Could we reword, such as `possible deployment patterns`?